blaze/compatibility.py

```diff
@@ -66,6 +80,7 @@ def boundmethod(func, instance):
     boundmethod = MethodType
 
     from itertools import zip_longest
+    from collections import abc as collections_abc
```

Comment:
i believe abstract collections were kept in the toplevel `collections` module for back compat, were they not?